protocol/messenger_mailserver.go

```diff
@@ -263,7 +263,7 @@ func getPrioritizedBatches() []int {
 	return []int{1, 5, 10}
 }
 
-func (m *Messenger) syncFilters(filters []*transport.Filter) (*MessengerResponse, error) {
+func (m *Messenger) syncFiltersFrom(filters []*transport.Filter, lastRequest uint32) (*MessengerResponse, error) {
```

Comment:
This is my attempt to have `syncFilters` provide a way to control how far we want to go back in time to sync from. 

If this is not a good way to do this, please let me know. Basically we need a way to say "Fetch all messages from store node from last 30 days"